ReSharper.FSharp/src/FSharp.Common/src/FcsReactorMonitor.fs

```diff
@@ -161,3 +156,48 @@ type FcsReactorMonitor
                 override __.Dispose () = match operations.TryRemove operationId with _ -> ()
                 override __.OperationName = sprintf "{%d}%s" operationId opName
             }
+
+        override __.OnOperationStart opName opArg = onOperationStart opName opArg
+        override __.OnOperationEnd() = onOperationEnd()
+
+
+[<ShellComponent>]
+type FcsReactorListener(logger: ILogger, reactorMonitor: IFcsReactorMonitor) =
+    interface IReactorListener with
+        override __.OnReactorPauseBeforeBackgroundWork pauseMillis =
+            logger.Verbose("Pausing before background work for {0:0.}ms", pauseMillis)
```

Comment:
Could it be `logger.Trace` instead? Would be great to hide most of FCS tracing unless it's turned on in the trace categories.

Reply:
Which messages do you want to be trace level? I really think keeping operation start and end as verbose is important for debugging. If a user doesn't have tracing enabled already it will be hard to debug issues that are tough to reproduce, as the logs won't really contain anything useful.

Reply:
Let's keep these two and and more back to verbose if it appears needed later?